hw/formal/tools/dvsim/common_fpv_cfg.hjson

```diff
@@ -4,6 +4,7 @@
 {
   sub_flow: fpv
   import_cfgs: ["{proj_root}/hw/formal/tools/dvsim/common_formal_cfg.hjson"]
+  synth_opt: ["{proj_root}/hw/formal/tools/jaspergold/synthesis_optimized.tcl"]
```

Comment:
Instead of this file, don't the lines setting `SYNTH_OPT` need to go in `hw/formal/tools/jaspergold`?

I think I'd probably be inclined to factor out the magic path too. Something like this, maybe?
```
  jg_tcl_dir: "{formal_root}/tools/{tool}"

  exports: [
    {JG_TCL_DIR: "{jg_tcl_dir}"},
    {COMMON_MSG_TCL_PATH: "{jg_tcl_dir}/jaspergold_common_message_process.tcl"}
  ]
```

Then we don't need to add `SYNTH_OPT` at all, and `fpv.tcl` can use `$env(JG_TCL_DIR)` instead.

Reply:
This makes sense, thanks.